src/analysis/seminorm.lean

```diff
@@ -395,7 +417,19 @@ end
 end has_scalar
 
 section module
-variables [norm_one_class 𝕜] [module 𝕜 E] (p : seminorm 𝕜 E)
+
+variables [module 𝕜 E]
+variables [add_comm_group E'] [module 𝕜 E']
+
+lemma comp_ball (p : seminorm 𝕜 E') (f : E →ₗ[𝕜] E') (x : E) (r : ℝ) :
```

Comment:
This should be `ball_comp` to match `ball_sup` (or `ball_sup` should be renamed instead)